src/styles/common.scss

```diff
@@ -236,6 +236,36 @@
     margin-bottom: 1%;
 }
 
+@media (max-width: $desktop-width) {
+    .textBanner {
+        padding: 70px 14%;
+    }
+
+    .textBannerSection {
+        margin-left: 2vw;
+        margin-right: 2vw;
```

Comment:
What is this for?

Reply:
To avoid this:
<img width="691" alt="Screenshot 2024-09-24 at 14 22 39" src="https://github.com/user-attachments/assets/b08192c7-ce5e-44b5-a942-c06cc4951f44">
Let me go around this

Reply:
And what exactly are we looking to avoid here?

Reply:
Actually I need that, but for mobile
<img width="1704" alt="Screenshot 2024-09-24 at 14 30 59" src="https://github.com/user-attachments/assets/374a149d-adc9-4e04-8429-f40c7e4c690b">
You can see on wix template padding 4%. 
We have 2vw on <main>. I need additional 2vw to get that 4%

Reply:
> And what exactly are we looking to avoid here?

That text is too close to margins. We need to have some space there